unityfiles/ProjectSettings/InputManager.asset

```diff
@@ -90,7 +90,7 @@ InputManager:
     descriptiveName: 
     descriptiveNegativeName: 
     negativeButton: 
-    positiveButton: space
+    positiveButton: up
```

Comment:
Er dette for å hoppe? Burde det ikke være med space?

Reply:
De sa at de skulle styre fra piltastene, regner da med at de også vil at hoppingen skal styres med piltastene, men vi kan spørre dem på mandag

Reply:
Det gir mening. Har du lagt det inn i spørsmåls listen?

Reply:
Dette bør vi på sikt ha som en konfigurasjonsmulighet også

Reply:
Skriver det ned i backloggen